packages/wallets/src/components/WalletsCarouselContent/WalletsCarouselContent.tsx

```diff
@@ -27,7 +28,13 @@ const WalletsCarouselContent: React.FC = () => {
         <div className='wallets-carousel-content' ref={wallet_embla_ref}>
             <div className='wallets-carousel-content__container'>
                 {wallet_accounts_list.map(wallet => (
-                    <WalletCard key={`${wallet.loginid}`} account={wallet} />
+                    <WalletGradientBackground
+                        key={`${wallet.loginid}`}
```

Comment:
@thisyahlen-deriv Please destructure the wallet  like what I mentioned in this file packages/wallets/src/components/WalletCard/WalletCard.tsx

Reply:
But im doing the mapping here. I still should de-structure it ?
I thought its cleaner to do the de-structure in the component itself when its receiving the props ?